.github/workflows/deploy.yml

```diff
@@ -52,17 +61,26 @@ jobs:
       - name: rcmdcheck
         run: |
           rcmdcheck::rcmdcheck(
-            error_on = "error", # TODO: switch back to "warning"
+            error_on = "error",
             check_dir = "check"
           )
         shell: Rscript {0}
         env:
           _R_CHECK_FORCE_SUGGESTS_: false
+      - name: rcmdcheck as CRAN
+        run: |
+          rcmdcheck::rcmdcheck(
+            args = "--as-cran",
```

Comment:
--no-manual to avoid the pdftatex errors. Not sure we need to build the full manual.